src/pages/shoppinglist.js

```diff
@@ -10,7 +11,7 @@ export default function ShoppingList() {
 				<title key="title">Shopping List</title>
 				<meta key="description" name="description" content="This is my project" />
 			</Helmet>
-			<h1>Shopping List</h1>
+			<StyledHeading>SHOPPING LIST</StyledHeading>
```

Comment:
Instead of capitalizing all your characters you could also use CSS: `text-transform: uppercase;`

Reply:
Good point, Mareike!
I implemented it.